float64[] joints
float64[] cartesians
float64[] moves
float64[] swings
float64 grippers
int32 framesID
int32 commandID
float64[] lspb
